docs/css/avatar.css

```diff
@@ -0,0 +1,42 @@
+.bat-avatar-xl {
+    height: 12rem;
+    width: 12rem;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+
+.bat-avatar-large {
+    height: 10rem;
+    width: 10rem;
+    border-radius: 50%;
```

Comment:
you are using border-radius 50% a lot of times , i would suggest put them in some css variable.

Reply:
Done